src/js/elements/mapchip.js

```diff
@@ -6,6 +6,7 @@ const wrapperClsName = 'content__map_current-display';
 const mapOptionsClass = '.map-options';
 const lightStart = 3;
 
+let title = '';
```

Comment:
why did you choose to define it outside of the class?

Reply:
Fixed it